infra.bs

```diff
@@ -1082,9 +1082,9 @@ providing a <a for=map>key</a> inside square brackets.
 <hr>
 
 <p>To <dfn export for=map lt="get|get the value">get the value of an entry</dfn> in an
-<a>ordered map</a> given a <a for=map>key</a> is to retrieve the <a for=map>value</a> of any
-existing <a for=map>entry</a> if the map <a for=map>contains</a> an entry with the given key, or
-to return nothing otherwise. We can also use the indexing syntax explained above.
+<a>ordered map</a> given a <a for=map>key</a>: if the map <a for=map>contains</a> an entry with the
+given key, then return the <a for=map>value</a> of the matching <a for=map>entry</a>; otherwise,
+return nothing. We can also use the indexing syntax explained above.
```

Comment:
If we're changing this, I think we should actually just assert that it contains it. This "nothing" value is confusing. For lists we say

> The index cannot be out-of-bounds, except when used with exists.

Reply:
Yeah, that sounds good.